os.go

```diff
@@ -34,3 +36,54 @@ func mkfs(dev, fstype string) error {
 	}
 	return nil
 }
+
+func chownIfEmpty(dir string, uid int, gid int) error {
+	files, err := ioutil.ReadDir(dir)
```

Comment:
sorry for annoying further, but `ioutil` functions are deprecated. Can we maybe use `os.ReadDir` instead? :sweat_smile: